code/screens/case-studies/components/introduceItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function IntroduceItem({ title, description, children }) {
+  return (
+    <div className="project-page-intro">
+      <h6>{title}</h6>
+      {description ? <p>{description}</p> : <div>{children}</div>}
```

Comment:
Why do u need to wrap children with div?

Reply:
removed